src/js/edu-benefits/1990e/reducer.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
```

Comment:
Yeah, I think that makes sense for reducer.js.

I think we may break Webpack's code splitting if we try to move routes.jsx into common, because it looks for paths in `require` calls inside `require.ensure`. If we pass that path as a parameter to a function, I don't think it'll work correctly. I'm not positive on that, though, so might be worth trying.